compiler/codegen/symbol.cpp

```diff
@@ -1213,7 +1213,13 @@ void TypeSymbol::codegenAggMetadata() {
 GenRet TypeSymbol::codegen() {
   GenInfo *info = gGenInfo;
   GenRet ret;
-  ret.chplType = typeInfo();
+  ret.chplType = type;
+
+  // Should not be code generating raw pointers
+  if (AggregateType* at = toAggregateType(type))
```

Comment:
Should probably add "by convention".
To me, all class types in codegen are "unmanaged", not "borrowed".